test/test_mps.py

```diff
@@ -1642,6 +1642,13 @@ def rotate_subset(data):
             mps_result = rotate_subset(mps_data)
             self.assertEqual(cpu_result, mps_result.to("cpu"))
 
+    # See https://github.com/pytorch/pytorch/issues/85967
+    def test_from_numpy_non_contiguous(self):
+        a = np.arange(9).reshape(3, 3)[:, :2]
+        t_cpu = torch.tensor(a, device="cpu")
```

Comment:
does this tensor constructor copy the passed data and makes always contig tensors? should this not be torch.as_tensor instead?

Reply:
Indeed it does, but that's the point: MPS copy operator did not handled this case correctly